package.json

```diff
@@ -8,7 +8,7 @@
     "nightwatch-single": "./node_modules/nightwatch/bin/nightwatch",
     "browser-tests": "NODE_ENV=test ./node_modules/karma/bin/karma start --browsers Chrome",
     "test-watch": "NODE_ENV=test ./node_modules/karma/bin/karma start --no-single-run",
-    "start": "NODE_ENV=development ./node_modules/webpack-dev-server/bin/webpack-dev-server.js --cache --colors --progress --port 3000 --host 0.0.0.0",
+    "start": "NODE_ENV=development ./node_modules/webpack-dev-server/bin/webpack-dev-server.js --cache --watch-poll --colors --progress --port 3000 --host 0.0.0.0",
```

Comment:
I've tested it out on my Mac (in an alpine linux Docker container, as that is our recommended development environment), and the idle CPU % for the Docker `hyperkit` process went from 2% to 200% on average.

I understand it's needed for Windows users, and this is something we want to accommodate, but I believe a better approach will be to make this configurable via an environment variable in `webpack.config.js`

Adding the following to the `devServer` config object should do it:

```
watchOptions: {
  poll: process.env.WEBPACK_POLL || false,
}
```

Reply:
Note: if running in Docker, make sure to add `WEBPACK_POLL:true` to the `environment` block for the `blip` service in your `docker-compose.yml` file.